sql/core/src/test/scala/org/apache/spark/sql/errors/QueryParsingErrorsSuite.scala

```diff
@@ -642,4 +642,92 @@ class QueryParsingErrorsSuite extends QueryTest with QueryErrorsSuiteBase {
           |^^^
           |""".stripMargin)
   }
+
+  test("UNSUPPORTED_FEATURE: cannot clean reserved namespace property") {
+    val sql = "CREATE NAMESPACE IF NOT EXISTS a.b.c WITH PROPERTIES ('location'='/home/user/db')"
+    validateParsingError(
+      sqlText = sql,
+      errorClass = "UNSUPPORTED_FEATURE",
+      errorSubClass = Some("CLEAN_RESERVED_NAMESPACE_PROPERTY"),
```

Comment:
I guess, the name of the error class might confuse users. Could you make it simpler CLEAN_RESERVED_NAMESPACE_PROPERTY -> SET_NAMESPACE_PROPERTY. So users should see:
```
[UNSUPPORTED_FEATURE.SET_NAMESPACE_PROPERTY]
```